plank/laravel-media-manager
# feat: #45 Label for media upload file size

## Description
- added file upload size label to mm-modal-add component.
-  added created life cycle method to handle when data should be fetched and updated.
- also created helper dir/with filter.js helper that displays the correct amount of bytes for file size.

## How to test

1.click go to encyclopedia & click media.
2.click the add/upload button (red circle with plus sign).
3. The modal that shows up should say at the bottom "Max file size: 20 MB"

![Group 1](https://user-images.githubusercontent.com/8687582/128879694-fb28fe13-28f4-417b-a4ce-8afe063c6c2f.png)

closes #45
